packages/fxa-auth-server/lib/payments/configuration/manager.ts

```diff
@@ -114,6 +114,26 @@ export class PaymentConfigManager {
     this.cancelPlanListener?.();
   }
 
+  /**
+   * Looks up a Firestore ProductConfig or PlanConfig document id based
+   * on the provided Stripe Product or Plan id.
+   */
+  public async getDocumentIdByStripeId(
+    stripeId: string
+  ): Promise<string | null> {
```

Comment:
I see `async` and `promise` in the declaration but I don't see an await in the function body.